tests/with_span_class_option.py

```diff
@@ -0,0 +1,17 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from unittest import main, TestCase
+
+
+class Test(TestCase):
+    def test_mdx_span_class_option(self):
+        from markdown import markdown
+        from mdx_unimoji import UnimojiExtension
+        text = 'I <3 you! Just kidding. :P'
```

Comment:
Um, okay. That test string is perfect because it checks the red style on the heart and the smiley faces, but sure.